lib/oli_web/templates/layout/authoring.html.leex

```diff
@@ -50,6 +50,11 @@
     <!-- Bootstrap typeahead https://github.com/bassjobsen/Bootstrap-3-Typeahead -->
     <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-3-typeahead/4.0.2/bootstrap3-typeahead.min.js" integrity="sha512-HWlJyU4ut5HkEj0QsK/IxBCY55n5ZpskyjVlAoV9Z7XQwwkqXoYdCIC93/htL3Gu5H3R4an/S0h2NXfbZk3g7w==" crossorigin="anonymous"></script>
 
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/citation-js"></script>
```

Comment:
what is the reason to include via script tag rather than https://www.npmjs.com/package/@citation-js/core

Reply:
That is the approach I tried first. I was unable to get the library to work as expected due to module loading errors resolving additional plugins required to support the range of citation formats required by the text-based editor. Adding the library to the global context resolved the issues